src/System.Text.Json/src/System/Text/Json/Node/JsonObject.cs

```diff
@@ -0,0 +1,465 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections;
+using System.Collections.Generic;
+
+namespace System.Text.Json
+{
+    /// <summary>
+    ///  Represents a JSON object.
```

Comment:
NIT: `Represents a JSON object.` -> `Represents a mutable JSON object.`

Reply:
Maybe it would be better to change `JsonNode` summary to "The base class that represents a single node within a modifiable JSON document.", because if I'm changing `JsonObject` summary I would probably also need to change JsonNumber etc. (but I can do both).